infra/stacks/auth.ts

```diff
@@ -0,0 +1,45 @@
+import heboCluster from "./cluster";
+import heboDatabase from "./db";
+import { allSecrets, isProd } from "./env";
+
+const authDomain = isProd ? "auth.hebo.ai" : `auth.${$app.stage}.hebo.ai`;
+export const authBaseUrl = `https://${authDomain}/v1`;
+const authPort = "3000";
+
+const heboAuth = new sst.aws.Service("HeboAuth", {
+  cluster: heboCluster,
+  architecture: "arm64",
+  cpu: isProd ? "1 vCPU" : "0.25 vCPU",
+  memory: isProd ? "2 GB" : "0.5 GB",
+  link: [heboDatabase, ...allSecrets],
+  image: {
+    context: ".",
+    dockerfile: "infra/docker/Dockerfile.auth",
+    tags: [authDomain],
+  },
+  environment: {
+    IS_REMOTE: $dev ? "false" : "true",
+    AUTH_BASE_URL: authBaseUrl,
+    AUTH_TRUSTED_ORIGINS: isProd
+      ? "https://console.hebo.ai"
```

Comment:
1/ Can you use a consoleUrl variable here (like you do use authUrl / apiUrl / ...) instead of redefining it?
2/ You are using TRUSTED_ORIGINS for different purposes (trustedOrigins in betterAuth configuration and for the e-mail template), given that, suggest to rename it to CONSOLE_URL.
3/ How does trustedOrigins work with API key authentication?